index.html

```diff
@@ -4,13 +4,152 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Global Enviromental Forum</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
-  <link src="style.css" ref="stylesheet">
+  <link href="style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/db2c8620f4.js" crossorigin="anonymous"></script>
+  
 </head>
 <body>
-  <script src="index.js"></script>    
+  <header id="header-container">
+    <nav class="nav-bar">
+      <div class="mobile"><img class="hamburguer-menu" src="./Images/Hamburguer_Menu.png"/></div>
+      <a href="index.html"><img class="forum-logo" src="./Images/Forum Logo.png"/></a>
+      <ul class="navbar-menu-list desktop">
+        <li><a class="nav-bar-menu-li" href="about.html">About</a></li>
+        <li><a class="nav-bar-menu-li" href="#program">Program</a></li>
+        <li><a class="nav-bar-menu-li"href="#">Join</a></li>
+        <li><a class="nav-bar-menu-li">Sponsor</a></li>
+        <li><a class="nav-bar-menu-li" href="#">News</a></li>
+        <li><a class="gef-campaing" href="#">GEF Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home-intro">
+      <div class="home-data-container">
+        <h2>"Protect your future enviroment!!"</h2>
+        <h1>CONSERVATION GLOBAL FORUM 2023</h1>
+      <div>
+        <p class="home-intro-parag-container">A massive kickoff where the most respectable Sustainability Desginers will define a Global plan to implement the use of biggest companies 5% of its pre-tax profits to protect our nature.</p>
+      </div>  
+      <div class="home-intro-location-container">
+        <h2>2023.05.24(WED) ~ 26(FRI)</h2>
+        <p>@ Chapelco Golf Resort, San Martín de Los Andes, Argentine Patagonia. </p>
+      </div>
+    </div>
+    </section>
+
+    <section id="program" class="main-program">
+      <h2>Main Program</h2>
+      <div class="separator"></div>
+
+      <div class="all-activities-container">
+        <div class="activity-container">
+          <div class="activity">
+          <div class="icon-container">
+            <img class="program-icon" src="./Images/Icons/planet-earth-svgrepo-com.svg" alt=""/>
```

Comment:
- [ ] Please make sure that `alt` attribute is not empty. Kindly apply this to the entire page 🤓⛏